packages/perseus/src/__tests__/renderer-api.test.tsx

```diff
@@ -46,8 +47,13 @@ describe("Perseus API", function () {
             // Act
             act(() => renderer.setInputValue(["input-number 1"], "5"));
 
+            const score = scorePerseusItemTesting(
+                inputNumber1Item.question,
+                renderer.getUserInputMap(),
+            );
+
             // Assert
-            expect(renderer).toHaveBeenAnsweredCorrectly();
+            expect(score).toHaveBeenAnsweredCorrectly();
```

Comment:
90% of the changes are just updating tests to score outside of Renderer and then using that score for the assertion.